{{cookiecutter.slug}}/backend/requirements.in

```diff
@@ -8,6 +8,9 @@ pytest-django
 pytest-xdist
 {% if cookiecutter.basic_authentication == "Yes, please!" %}
 dj_rest_auth
-django-allauth
+django-allauth<65.3
```

Comment:
Necessary not to go too high here, as otherwise settings for SAML clash with settings for allauth.